debugtools/DDR_VM/data/superset-constants.dat

```diff
@@ -4597,7 +4597,7 @@ C|allocation_category_mixed
 S|MM_LargeObjectAllocateStats|MM_LargeObjectAllocateStatsPointer|MM_Base
 C|MAX_FREE_ENTRY_COUNTERS_PER_FREQ_ALLOC_SIZE
 C|VERY_LARGE_ENTRY_POOL_SIZE_FOR_THREAD
-S|MM_LightweightNonReentrantReaderWriterLock|MM_LightweightNonReentrantReaderWriterLockPointer|MM_BaseNonVirtual
+S|MM_LightweightNonReentrantRWLock|MM_LightweightNonReentrantRWLockPointer|MM_BaseNonVirtual
```

Comment:
Additions should **not** be made to this file (`MM_LightweightNonReentrantRWLock` was not previously present).

Reply:
Fixed via #13417.